backend/osmosewebsite/models/trap.py

```diff
@@ -0,0 +1,28 @@
+"""Trap-related models"""
+
+from django.db import models
+from django.contrib.postgres import fields
+from tinymce.models import HTMLField
+from .team_member import TeamMember
+
+
+class Trap(models.Model):
+    """
+    Table containing Trap will be used on the website
+    """
+
+    firstname = models.CharField(max_length=100, default="DefaultFirstName")
+    lastname = models.CharField(max_length=100, blank=True, null=True)
+    title = models.CharField(max_length=255, unique=True)
+    intro = models.CharField(max_length=255)
+    date = models.DateField(null=True, blank=True)
+    thumbnail = models.URLField(default="")
+    osmose_member_authors = models.ManyToManyField(TeamMember, blank=True)
```

Comment:
How is it used?

Reply:
It is used for the presenter's first name and last name.

Reply:
After our exchange, it has been removed